packages/@sanity/default-login/src/util/getProviders.js

```diff
@@ -20,7 +20,8 @@ export default function getProviders() {
         // login options through the config. These shouldn't be treated as custom login providers
         // which require the third-party login feature, but as the official provider
         const isOfficial = providers.some((official) => official.url === provider.url)
-        return {...provider, custom: !isOfficial, supported: isOfficial || thirdPartyLogin}
+        const isSupported = isOfficial || thirdPartyLogin || (sso && sso[provider.type])
```

Comment:
Would it be better to do an explicit check? Or if we want to reduce the configuration surface we could do something like `sso && Object.values(sso).some(Boolean)`?

Reply:
👍